xc/xenops_server_xen.ml

```diff
@@ -733,6 +755,15 @@ module HOST = struct
     with_xc_and_xs
       (fun xc xs ->
          let open Xenctrl in
+
+         let xen_sysctl_cpu_policy = Hashtbl.create 6 in
+         Hashtbl.add xen_sysctl_cpu_policy "XEN_SYSCTL_cpu_policy_raw" 0;
```

Comment:
The keys for the hash table are used in multiple places. I'd bind them to names and uses them to avoid typos.